with-crm-components/package.json

```diff
@@ -4,7 +4,7 @@
   "description": "UI extension example using CRM components",
   "main": "index.js",
   "scripts": {
-    "postinstall": "cd ./src/app/extensions/ && npm install",
+    "postinstall": "cd ./src/app/extensions/ && npm install && cd ../app.functions && npm install",
```

Comment:
This command was working, it just wasn't running the install in the functions directory so I updated it to match.